### PR TITLE
adding the known issue for observability after upgrade

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/14562

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -398,6 +398,27 @@ Due to unauthorized access to the `managedcluster` resource, you might receive t
...
 
+[#mco-error-during-upgrade]
+=== Deployment error with MultiClusterObservability CR after upgrade
+
+When you upgrade from {product-version-prev}.x to {product-version} and deploy observability, you might receive the following error statement in the MultiClusterObservability CR: `Failed to find expected deployment observability-grafana`. Complete the following steps to fix this issue:
+
+. Delete the MultiClusterObservability CR with the following command:
++
+----
+oc delete mco
```
Comment (marcolan018) [inline release_notes/known_issues.adoc]: correct command should be 

oc delete mco observability

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -398,6 +398,27 @@ Due to unauthorized access to the `managedcluster` resource, you might receive t
...
+----
+
+. Restart the `multicluster-observability-operator` pod by running the following command:
++
+----
+filler command here
```
Comment (marcolan018) [inline release_notes/known_issues.adoc]: command should be

oc delete po {POD_NAME} -n open-cluster-management

{POD_NAME} is the name of `multicluster-observability-operator` pod

**Comment by dockerymick** [comment]
Thanks Marco! @marcolan018

**Pull request merged**